docs/flow/config-language/files.md

```diff
@@ -6,3 +6,18 @@ weight: 100
 ---
 
 # Files
+River files are plaintext files with the `.river` file extension. Each River
+file may be referred to as a 'configuration file', or a 'River configuration'.
+
+River files are required to be UTF-8 encoded, and are permitted to contain
+Unicode characters. River files can use both Unix-style line endings (LF) and
+Windows-style line endings (CRLF), but formatters may replace all line endings
+with Unix-style ones.
+
+There is experimental support for River in
+[vim](https://github.com/rfratto/vim-river) and in
+[VSCode](https://github.com/rfratto/vscode-river).
+
+To learn more about the basic elements that form a River configuration file,
+check out the [Syntax]({{< relref "../../controller-config/logging.md" >}})
+guide next.
```

Comment:
nit: we shouldn't have to guide people through paragraphs, they can use the sidebar to navigate and expect that they should be reading top to bottom.

Reply:
I personally find it useful to have another way of saying "hey, click here to continue reading" instead of expecting that they're following the sidebar, especially in mobile devices or resized browser windows where the sidebar might be hidden altogether.

I can still remove it if you still think it looks out of place, just let me know ^^